test/unit/ajax.js

```diff
@@ -2840,6 +2840,32 @@ if ( typeof window.ArrayBuffer === "undefined" || typeof new XMLHttpRequest().re
 		} );
 	} );
 
+	QUnit.test( "jQuery.append with crossOrigin attribute", function( assert ) {
+		var done = assert.async();
+		assert.expect( 1 );
+
+		var timeout = null;
+
+		Globals.register( "corsCallback" );
+		window.corsCallback = function( response ) {
+			assert.ok( response === true, "Cors" );
+			window.clearTimeout( timeout );
+			done();
+		};
+
+		var src = baseURL + "mock.php?action=cors";
+
+		// To simulate a cross-origin request
+		src = src.replace( "localhost", "127.0.0.1" );
+		var html = "<script type=\"text/javascript\" src=\"" + src + "\" crossorigin=\"anonymous\"><\/script>";
+
+		jQuery( document.body ).append( html );
+		timeout = window.setTimeout( function() {
+			assert.ok( false, "Cors" );
```

Comment:
This assertion fires for me in Safari 13, Edge 18 & IE 11. It correctly doesn't fire only in Firefox 72 & Chrome 79.

We need to figure out why it fails in other browsers. For IE we might have to skip the test as it doesn't support the crossorigin attribute (by switching from `QUnit.test` to `QUnit.testUnlessIE`). Other browsers should pass it, though.

From my recent investigation: it seems the tests pass in Safari in Karma but not via the PHP setup. On the other hand, they succeed in both modes in Chrome & Firefox.